docs/core/entity/camera.md

```diff
@@ -20,7 +20,7 @@ Properties should always only return information from memory and not do I/O (lik
 | brand                    | str   | `None`  | The brand (manufacturer) of the camera.                                                                                 |
 | model                    | str   | `None`  | The model of the camera.                                                                                                |
 | frame_interval           | float | 0.5     | The interval between frames of the stream.                                                                              |
-| frontend_stream_type     | str   | None    | Used with `SUPPORT_STREAM` to tell the frontend which type of stream to use (`STREAM_TYPE_HLS` or `STREAM_TYPE_WEBRTC`) |
+| frontend_stream_type     | str   | None    | Used with `CameraEntityFeature.STREAM` to tell the frontend which type of stream to use (`StreamType.HLS` or `StreamType.WEB_RTC`)  |
```

Comment:
I must be missing something but I don't find the `StreamType` enum anywhere

Reply:
That is quite possible, as the PR for that is open and linked to this PR :)

Reply:
So I spent 5 minutes searching for this instead of 2 seconds reading the PR description 😂